### DISCUSSION TITLE
提问帖 想请问Disqus评论怎么使用呢

### DISCUSSION THREAD

**Discussion by Poliki** [CaiJimmy/hugo-theme-stack#254] [Q&A]
如题

**Comment by zhixuan2333** [comment]
https://docs.stack.jimmycai.com/zh/configuration/#comments

**Reply by gleb-kun** [reply]
Hi, can you please update this link?

**Reply by zhixuan2333** [reply]
Of course!

https://stack.jimmycai.com/config/comments

**Comment by Poliki** [comment]
邮件已收到，非常感谢！
